scripts/install-snapshot.sh

```diff
@@ -2,12 +2,14 @@
 
 # This script can be used to install/delete snapshotcontroller and snapshot beta CRD
 
-SNAPSHOT_VERSION=${SNAPSHOT_VERSION:-"v2.1.1"}
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+# shellcheck source=build.env
+[ ! -e "${SCRIPT_DIR}"/../build.env ] || source "${SCRIPT_DIR}"/../build.env
 
-SCRIPT_DIR="$(dirname "${0}")"
+SNAPSHOT_VERSION=${SNAPSHOT_VERSION:-"v3.0.0"}
 
 TEMP_DIR="$(mktemp -d)"
-SNAPSHOTTER_URL="https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOT_VERSION}"
+SNAPSHOTTER_URL="https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${SNAPSHOT_VERSION}/client"
```

Comment:
changing it here won't work directory is changed only for CRD, not for RBAC

Reply:
Ah! ok, I will fix it. Thanks!

Reply:
done now. Thanks, Madhu.